README.md

```diff
@@ -287,13 +287,13 @@ The method form takes parameters
 `auth(username, password, sendImmediately, bearer)`.
 
 `sendImmediately` defaults to `true`, which causes a basic or bearer
-authentication header to be sent.  If `sendImmediately` is `false`, then
```

Comment:
Thanks for the context, hard to get caught up when Github minimizes everything that was previously said.